lock-farm/contracts/LockFarm.sol

```diff
@@ -191,14 +191,16 @@ contract LockFarm is
     function getFnfts(address owner)
         external
         view
-        returns (FNFTInfo[] memory infos)
+        returns (uint256[] memory ids, FNFTInfo[] memory infos)
```

Comment:
Instead of returning another ids array, would it be a bad idea to create a field in the struct FNFTInfo to include the fnftID?